isdf/datasets/dataset.py

```diff
@@ -242,10 +294,17 @@ def __init__(
         torch.multiprocessing.set_start_method('spawn', force=True)
         self.queue = torch.multiprocessing.Queue(maxsize=1)
 
-        process = torch.multiprocessing.Process(
-            target=node.iSDFNode,
-            args=(self.queue, crop),
-        )
+        if dataset_format is not None:
```

Comment:
We should change the name of `dataset_format` to `extrinsic_calib` in this class. Looks like `dataset_format` is reading `self.ext_calib` when the class is instantiated, and this is None for all datasets apart from the realsense_franka. We should make that more clear in the class definition.

Reply:
Thanks for this! Changes the argument name to `extrinsic_calib` in `ROSSubscriber`.